utilities/openemr-env-installer/ubuntu/openemr-ubuntu-installer

```diff
@@ -0,0 +1,146 @@
+if [ ! -d ${CODE_LOCATION} ]
+then
+    mkdir -p ${CODE_LOCATION}
+    git_clone_function $1 $2
+elif [ ! -d ${CODE_LOCATION}/openemr ]
+then
+    git_clone_function $1 $2
+else
+    echo -e "\033[30m===>\033[0m \033[32mOpenEMR repo is already downloaded. \033[0m"
+    echo ''
+fi
+
+# Install docker
+sudo systemctl start docker &>/dev/null
+if [ $? -ne 0 ]
+then
+ 	echo -e "\033[30m===>\033[0m \033[32mUpdate the apt package index and install packages to allow apt to use a repository over HTTPS... \033[0m"
+	echo ''
+	sudo apt-get update -y
+	sudo apt-get install apt-transport-https ca-certificates curl gnupg-agent software-properties-common -y
+	echo ''
+	echo -e "\033[30m===>\033[0m \033[32mAdding the Docker official GPG key... \033[0m"
+	sudo curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+	echo ''
+	echo -e "\033[30m===>\033[0m \033[32mSetting up the stable repository... \033[0m"
+	sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+	echo ''
+	echo -e "\033[30m===>\033[0m \033[32mInstalling docker engine... \033[0m"
+	echo ''
+	sudo apt-get update -y
+	sudo apt-get install docker-ce docker-ce-cli containerd.io -y
+	sudo systemctl start docker
+	sudo systemctl enable docker
+	echo ''
+else
+	echo -e "\033[30m===>\033[0m \033[32mDocker is already installed. \033[0m"
+	sudo systemctl enable docker
+	echo ''
+fi
+
+# Install docker-compose
+which docker-compose &>/dev/null
+if [ $? -ne 0 ]
+then
+	echo -e "\033[30m===>\033[0m \033[32mInstalling docker-compose... \033[0m"
+	echo ''
+	sudo curl -L "https://github.com/docker/compose/releases/download/1.26.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+	sudo chmod +x /usr/local/bin/docker-compose
+	echo ''
+else
+	echo -e "\033[30m===>\033[0m \033[32mDocker-compose is already installed. \033[0m"
+	echo ''
+fi
+
+# Install openemr-cmd
+which openemr-cmd &>/dev/null
+if [ $? -ne 0 ]
+then
+	echo -e "\033[30m===>\033[0m \033[32mInstalling openemr-cmd... \033[0m"
+    echo ''
+	sudo curl -L https://raw.githubusercontent.com/openemr/openemr-devops/master/utilities/openemr-cmd/openemr-cmd -o /usr/local/bin/openemr-cmd
+	sudo chmod +x /usr/local/bin/openemr-cmd
+	echo ''
+else
+	echo -e "\033[30m===>\033[0m \033[32mOpenEMR-cmd is already installed. \033[0m"
+	echo ''
+fi
+
+# Get the value after started docker
+DOCKER_STATUS=$(sudo systemctl is-active docker)
+# Check the installation result
+echo '*********Quick Check*********'
+git --version
+docker --version
+docker-compose --version
+openemr-cmd --version
+echo -e "\033[36mThe docker status: \033[0m\033[32m${DOCKER_STATUS} \033[0m"
+echo '*****************************'
+echo ''
+
+# Startup env
+echo '*****************************Startup Env*****************************'
+echo -e "\033[34m==>\033[0mRun the below command to startup \033[33measy env\033[0m."
+echo -e "\033[32mcd ${CODE_LOCATION}/openemr && sudo docker-compose up\033[0m"
```

Comment:
dang @reidliu41 , I am guessing that pretty soon you will make a script that sends OpenEMR to the moon :)
Very nice stuff! for ubuntu/debian would set it up so that sudo is not needed on the docker and docker-compose commands. see step 2 here:
https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-on-ubuntu-18-04

Reply:
this then allows following commands:
```
echo -e "\033[32mcd ${CODE_LOCATION}/openemr && openemr-cmd up\033[0m"
```
and
```
echo -e "\033[32mcd ${CODE_LOCATION}/openemr/contrib/util/docker && openemr-cmd up\033[0m"
```

Reply:
@bradymiller, yeap, haha, just want to make it easier. thanks for your suggestion.